File: src/2652.sum-multiples.2752/solution.ts
function sumOfMultiples (n: number): number {
    let result = 0;
    for (let i = 3; i <= n; i++) {
        if (i % 3 === 0 || i % 5 === 0 || i % 7 === 0) {
            result += i;
        }
    }
    return result;
}

File: src/2652.sum-multiples.2752/solution.java
class Solution {
    public int sumOfMultiples(int n) {
        int result = 0;
        for (int i = 2; i <= n; i++) {
            if (i % 3 == 0 || i % 5 == 0 || i % 7 == 0) {
                result += i;
            }
        }
        return result;
    }
}